lib/widgets/components/receipt/dropdown_menu.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:coffeecard/base/style/colors.dart';
+import 'package:coffeecard/base/style/text_styles.dart';
+import 'package:coffeecard/blocs/receipt/receipt_cubit.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class FilterDropdown extends StatelessWidget {
```

Comment:
Did the design change on this? The one I have acces to looks more like the old dropdown

Reply:
That's reasonable, just wanted to know the thought process behind it